addons/website_slides/views/website_slides_templates_homepage.xml

```diff
@@ -170,13 +172,25 @@
     <t t-set="body_classname" t-value="'o_wslides_body'"/>
     <t t-call="website.layout">
         <div id="wrap" class="wrap o_wslides_wrap">
-            <section class="s_banner bg-900" style="background-image: url(&quot;/website_slides/static/src/img/banner_default_all.svg&quot;); background-size: cover; background-position: 80% 20%" data-snippet="s_banner">
-                <div class="container py-5">
-                    <h1 t-if="search_my" class="display-3 mb-0">My Courses</h1>
-                    <h1 t-elif="search_slide_category=='certification'" class="display-3 mb-0">Certifications</h1>
-                    <h1 t-else="" class="display-3 mb-0">All Courses</h1>
-                </div>
-            </section>
+            <!-- Almost duplicated lines required to allow customization with the website editor !-->
+            <div class="oe_structure oe_empty" t-if="search_my">
+                <section class="s_banner bg-900" data-snippet="s_banner"
+                         style="background-image: url(&quot;/website_slides/static/src/img/banner_default_all.svg&quot;); background-size: cover; background-position: 80% 20%">
+                    <div class="container py-5"><h1 class="display-3 mb-0">My Courses</h1></div>
+                </section>
+            </div>
+            <div class="oe_structure oe_empty" t-elif="search_slide_category=='certification'">
```

Comment:
We usually space the condition a bit, so `t-elif="search_slide_category == 'certification'"`
(I know you copy/pasted but let's change it while we are working on it).